client.go

```diff
@@ -97,26 +95,17 @@ func (c *Client) client() *http.Client {
 	return client
 }
 
-func (c *Client) convertOptsToURLValues(v interface{}) url.Values {
-	values, _ := query.Values(v)
-	return values
-}
```

Comment:
Hmm, so this function never works and we never convert the params into query URL for GET/DELETE requests?

Reply:
I was referring to the `convertOptsToURLValues` function, which seems to have never worked before this PR? We use `convertOptsToURLValues` in the GET request body it was still sending the request data in the JSON body?

Reply:
It was used sporadically and inconsistently in upstream functions, which was likely what was causing some GET requests to be using body. Its new location ensures it will be applied to all GET requests.